server/build.gradle

```diff
@@ -64,6 +68,7 @@ configurations {
 }
 
 dependencies {
+    implementation "org.apache.xmlbeans:xmlbeans:${versions.xmlbeans}"
     implementation "org.springframework.boot:spring-boot-starter-web"
```

Comment:
should these springboot versions be pinned to 3.1.4 too?

Reply:
it's enough to set the version of springboot in plugins block